### DISCUSSION TITLE
Feature request: add information about data received and transmitted

### DISCUSSION THREAD

**Discussion by random3456345** [evilsocket/opensnitch#1244] [Ideas]
I think it would be useful  to see the data transferred by applications, addresses and so on.

**Comment by slynobody** [comment]
i do it with the plasmoid of sysemmonitor/network-speed currently. but seconding this (would be more specific and little-snitch-like)

**Reply by gustavo-iniguez-goya** [reply]
If you mean transferred bytes, it's almost done: https://github.com/evilsocket/opensnitch/commit/4cdf709606972f9098166664b3bff1786a4187c0 / https://github.com/evilsocket/opensnitch/issues/1081

But I need to rewrite it to collect stats by connections, instead of applications. Later we can group by app.
